parts/kubernetesmastercustomscript.sh

```diff
@@ -169,12 +169,6 @@ function configNetworkPolicy() {
     fi
 }
 
-function ensureEtcd() {
```

Comment:
Yeah that's my impression as well on discussing this further with the Linux team that CSE does not run until cloud init finishes. 

Plus regarding:

> now that this is removed from the script, and since CSE is started in parallel, what is the synchronization point between the etc setup finishing, and this script starting?

My understanding is that this was never mean for synchronization in the first place. But just to be sure I will add the check.